tests/helper/helper.go

```diff
@@ -796,3 +796,14 @@ func generateCA(t *testing.T) {
 		require.NoErrorf(t, err, "error closing custom CA key file- %s", err)
 	}
 }
+
+func CheckKubectlGetResult(t *testing.T, kind string, name string, namespace string, otherparameter string, expected string) {
```

Comment:
could you please add a short doc comment here on what this function does and returns?